opencti-platform/opencti-front/src/utils/Store.js

```diff
@@ -11,7 +11,12 @@ export const isNodeInConnection = (payload, conn) => {
 export const insertNode = (store, key, filters, rootField) => {
   // Build record ids
   const record = store.get(store.getRoot().getDataID());
-  const conn = ConnectionHandler.getConnection(record, key, filters);
+
+  // TODO IT BREAKS CONNECTION ?
```

Comment:
You can change that and put the explanation we talk about.

Reply:
Done